src/components/SignUp/ActorInfo1.tsx

```diff
@@ -32,4 +213,30 @@ const ImageCol = styled(Col)`
   max-width: 100%;
 `;
 
+const CAGLabel = styled(Form.Label)`
+  color: ${colors.mainFont};
+  font-family: ${fonts.mainFont};
+  font-size: 20px;
+`;
+
+const PrivateSpan = styled.span`
+  font-size: 16px;
+  color: ${colors.primary};
+  font-weight: bold;
+`;
+
+const PrivacyPar = styled.p`
+  color: ${colors.mainFont};
+  font-family: ${fonts.lora};
+  font-size: 18px;
+  letter-spacing: 0px;
+  margin-top: 17px;
+
+  link {
```

Comment:
`link` isn't the underlying tag, `a` is. If you also don't need to change the color of it, you don't need to have the style block here for it at all.

Reply:
Makes sense. The flow has the link being red, but I just thought for consistency I should keep it the same color. What do you think?

Reply:
@aminnoah yup! That sounds good. So you can remove this link {} block altogether then 👍🏻